Library/views.py

```diff
@@ -30,7 +36,69 @@ def post(self, request):
             user = authenticate(email=email, password=password)
 
             if user is not None:
-                return Response({'msg': 'Login Successful'})
+                refresh = CustomTokenObtainPairSerializer.get_token(user)
+                return Response({'refresh': str(refresh),
+                                 'access': str(refresh.access_token), 'msg': 'Login Successful'})
             else:

```

Comment:
we don't need else, as if the "if" condition evaluate to false the code outside the if condition will be executed.